server_release_template/web_gui/js/app/setupWizard.js

```diff
@@ -7,7 +7,39 @@ define([
 ], function (_, driverList, wizardTemplate, i18n) {
     return function (alvrSettings) {
 
+        class GPU {
+            constructor() {
+                const gl = document.createElement('canvas').getContext('webgl');
+                const debugInfo = gl.getExtension("WEBGL_debug_renderer_info");
+                var rawGPUInfo = gl.getParameter(debugInfo.UNMASKED_RENDERER_WEBGL);
+
+                this.fullName = rawGPUInfo.match(/((NVIDIA|AMD|Intel)[^\d]*[^\s]+)/)[0];
+                [this.dev, this.name] = this.fullName.split(/(?<=^\S+)\s/);
+            }
+        }
+
+        function CheckGPUSupport(name) {
+            if (/(GeForce ((8[3-9][0-9]|9[0-3][0-9]|94[0-5])[AM]|GT 1030|GTX 9([2-3][0-9]|40)MX|MX(110|130|1[5-9][0-9]|2[0-9][0-9]|3[0-2][0-9]|330|350|450))|Radeon (((VIVO|[2-9][0-9][0-9][0-9]) ?\S*)|VE|LE|X(1?[0-9][0-5]0)))/.test(name)) { //Kill me pls
+                return '🔴 '+i18n.GPUUnsupported;
+            } else {
+                return '🟢 '+i18n.GPUSupported;
+            }
+        }
 
+        function CheckGPUSSupport(GPU) {
+            switch (GPU.dev) {
+                case "Intel(R)":
+                    return '🔴 '+i18n.GPUUnsupported;
+                    break;
+                case "NVIDIA":
+                case "AMD":
+                    return CheckGPUSupport(GPU.name);
+                    break;
+                default:
+                    return "🟣 WT..?! You wanna say you have no GPU? Are you playing on potato?"
```

Comment:
I know this is funny but actually users that see this message may (most probably) have their GPU disconnected. Also add this as a localization string.